src/pages/connect-child-to-cabin/connect-child-to-cabin.html

```diff
@@ -38,7 +38,7 @@ <h1>Kinderen in hut{{hutNr?' ' + hutNr:''}}:</h1>
 				'-' }})
 			</button>
 		</div>
-		<button ion-button full (click)="showAddModal()" [ngClass]='hutNr > 99?"shown":""'
+		<button ion-button full (click)="showAddModal()" [ngClass]='hutNr && hutNr.length === 3 ? "shown":""'
```

Comment:
maar als hutNr undefined is, krijg je dan niet "cannot read property length of undefined"?

Reply:
Daarom doe ik hutNr **&&** hutNr.length === 3
Als hutNr undefined is komt ie al terug met false voordat ie het tweede deel van de statement gaat checken

Reply:
ah nice, vanilla JS geeft dan juist wél een foutmelding. ik ga zo testen op android